packages/explat-client-react-helpers/src/index.tsx

```diff
@@ -64,18 +64,18 @@ export default function createExPlatClientReactHelpers(
 			...providedOptions,
 		};
 
+		// Manual updates to ensure rerendering when we want it:
+		// https://legacy.reactjs.org/docs/hooks-faq.html#is-there-something-like-forceupdate
+		// eslint-disable-next-line @typescript-eslint/no-unused-vars
+		const [ _ignored, forceUpdate ] = useReducer( ( x ) => x + 1, 0 );
```

Comment:
I think you can avoid needing to disable ESlint by just not using a variable.

```suggestion
		const [ , forceUpdate ] = useReducer( ( x ) => x + 1, 0 );
```